pyomo/contrib/doe/fim_doe_tutorial.ipynb

```diff
@@ -0,0 +1,1814 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Reactor Kinetics Example \n",
+    "\n",
+    "Jialu Wang (jwang44@nd.edu) and Alex Dowling (adowling@nd.edu)\n",
+    "\n",
+    "University of Notre Dame\n",
+    "\n",
+    "This notebook conducts design of experiments for a reactor kinetics experiment with the Pyomo.DOE.\n",
+    "    "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 0: Import Pyomo and Pyomo.DOE module"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import pyomo.environ as pyo\n",
+    "import doe as doe\n",
+    "import pyomo.common.unittest as unittest\n",
+    "from measurements import Measurements"
```

Comment:
Should these import statements be updated to reflect the other changes from today?